Makefile

```diff
@@ -52,6 +52,12 @@ run-website-staging:
 	cd gcp/website/blog && hugo --buildFuture -d ../dist/static/blog
 	cd gcp/website && $(install-cmd) && GOOGLE_CLOUD_PROJECT=oss-vdb-test $(run-cmd) python main.py
 
+
+run-website-emulator:
+	cd gcp/website/frontend3 && npm install && npm run build
+	cd gcp/website/blog && hugo --buildFuture -d ../dist/static/blog
+	cd gcp/website && $(install-cmd) && GOOGLE_CLOUD_PROJECT=test-osv DATASTORE_EMULATOR_PORT=5002 $(run-cmd) python frontend_emulator.py
```

Comment:
nit: setting `GOOGLE_CLOUD_PROJECT` is probably unnecessary, since `start_datastore_emulator` should do this.